CentralizedConsentAPI/README.md

```diff
@@ -0,0 +1,184 @@
+# Centralized Consent API
+
+## Authors:
+
+- Kate Cheney
```

Comment:
Your name should be a link, either to your GitHub profile, website, email address, something like that.